spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/WebOperationService.java

```diff
@@ -260,6 +268,23 @@ private HttpResponse handleResult(ServiceRequestContext ctx,
         return HttpResponse.of(status, contentType, body.toString());
     }
 
+    @Nullable
+    private static MediaType toMediaType(@Nullable MimeType mimeType) {
+        if (mimeType == null) {
+            return null;
+        }
+        final MediaType mediaType = MediaType.create(mimeType.getType(), mimeType.getSubtype());
+        final Map<String, String> parameters = mimeType.getParameters();
+        if (parameters.isEmpty()) {
+            return mediaType;
+        }
+        return mediaType.withParameters(
+                parameters.entrySet()
+                          .stream()
+                          .collect(toImmutableMap(Map.Entry::getKey,
+                                                  entry -> ImmutableSet.of(entry.getValue()))));
```

Comment:
Not related to this PR, this looks inconvenient. How about adding a new variant of `withParameters` to directly inject parameters.
```java
// Map<String, String> can not be used due to type erasure.
public MediaType withParameters(Iterable<? extends Entry<String, String>> parameters) {
    ...
}

return mediaType.withParameters(parameteres.entrySet());
```

Reply:
That's a good suggestion. 👍

Reply:
It seems like we cannot add the method because we already have a method that takes a `Map`. Could you check it, please?
`Cannot resolve method 'withParameters(java.util.Map<java.lang.String,java.lang.String>)'`

Reply:
Ah now I realise that the `Map` does not extend the `Iterable`. 😅

Reply:
We forked this source from Guava and I think it's pretty common to accept `Map<String, ? extends Iterable<String>>` type. Let me add another static method later if there's a demand for `Map<String, String>`.

Reply:
Guava users could easily build `Map<String, ? extends Iterable<String>>` using `ImmutableListMultimap.Builder`. However, it is pretty painful to create `Map<String, ? extends Iterable<String>>` using JDK. 
Anyway, let's add it later there is a demain.